py-wacz/wacz/util.py

```diff
@@ -0,0 +1,5 @@
+import hashlib 
+
+def support_hash_file(data):
+    '''Hashes the passed content using sha224'''
+    hashlib.sha224(data).hexdigest()
```

Comment:
Just wondering, do we want to standardize on sha-224?
I've been generating sha-256 in other places (warcio.js), maybe should just use sha-256? Or provide options?